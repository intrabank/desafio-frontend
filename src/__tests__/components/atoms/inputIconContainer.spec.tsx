import '@testing-library/jest-dom';
import renderer from 'react-test-renderer';
import InputIconContainer from '../../../components/atoms/inputIconContainer';

describe('InputIconContainer', () => {
	it('should render card component', () => {
		const tree = renderer
			.create(
				<InputIconContainer>
					<i></i>
				</InputIconContainer>
			)
			.toJSON();
		expect(tree).toMatchSnapshot();
	});
});
