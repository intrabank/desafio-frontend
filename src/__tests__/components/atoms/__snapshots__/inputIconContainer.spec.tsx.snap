// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InputIconContainer should render card component 1`] = `
<div
  className="c-bdGibU"
>
  <i />
</div>
`;
